packages/plugin-html-button-response/src/index.ts

```diff
@@ -100,10 +103,21 @@ class HtmlButtonResponsePlugin implements JsPsychPlugin<Info> {
     buttonGroupElement.id = "jspsych-html-button-response-btngroup";
     if (trial.button_layout === "grid") {
       buttonGroupElement.classList.add("jspsych-btn-group-grid");
-      let n_cols =
-        trial.button_cols === "auto" ? trial.choices.length : parseInt(trial.button_cols);
+      if (trial.button_rows === null && trial.button_cols === null) {
+        throw new Error(
+          "You must specify the number of rows or columns when using the grid layout."
```

Comment:
I tend to misunderstand this message: Folks don't need to specify any of them, as long as they don't pass `null` to `trial.button_rows` explicitly. Maybe 
```suggestion
          "You cannot set `button_rows` to `null` without providing a value for `button_cols`."
```
?